frontend/src/app/libs/container/services/localization.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import { Locale } from '../models/locale';
+import { Localization } from '../models/localization';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalizationService {
+
+	private _currentLocale: Locale = 	{ name: 'EN', code: 'en' };
+	private _localizations: Map<Locale, Localization> = new Map([
```

Comment:
make this service using json files, this file quickly become too big, for .ts file